server-spi-private/src/main/java/org/keycloak/migration/migrators/MigrateTo9_0_0.java

```diff
@@ -83,4 +89,71 @@ protected void addAccountConsoleClient(RealmModel realm) {
             client.addProtocolMapper(audienceMapper);
         }
     }
+
+
+    protected void migrateAuthenticationFlowsWithAlternativeRequirements(RealmModel realm) {
+        for (AuthenticationFlowModel flow : realm.getAuthenticationFlows()) {
```

Comment:
Is this also returing non-root flows (e.g. OTP sub flow of the browser flow)?

Reply:
yes, it is returning also non-root flows.